cypress/e2e/05-get_result_from_patient_link.cy.js

```diff
@@ -19,7 +19,6 @@ const {
 const specRunName = "spec05";
 const currentSpecRunVersionName = `${testNumber()}-cypress-${specRunName}`;
 
-loginHooks();
 describe("Getting a test result from a patient link", () => {
```

Comment:
when I run this locally, I am getting this error
<img width="827" alt="Screenshot 2024-03-20 at 13 50 10" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/a21fd199-e8a3-436a-b3f2-ca1620dfd5bc">

Reply:
thank you for addressing this!